hoomd/mpcd/ATCollisionMethodGPU.h

```diff
@@ -34,10 +34,10 @@ class PYBIND11_EXPORT ATCollisionMethodGPU : public mpcd::ATCollisionMethod
 
     protected:
     //! Draw velocities for particles in each cell on the GPU
-    virtual void drawVelocities(uint64_t timestep);
+    virtual void drawVelocities(uint64_t timestep) override;
```

Comment:
```suggestion
    void drawVelocities(uint64_t timestep) override;
```
I don't run clang-tidy on hoomd, but it would tell us to use `override` instead of `virtual`. It is the modern way (tm).